source/Camera/Basic/CaptureHDRLoop/CaptureHDRLoop.cpp

```diff
@@ -0,0 +1,49 @@
+/*
+This example shows how to acquire HDR images from the Zivid camera in a loop
+(while actively changing some HDR settings).
+*/
+
+#include <Zivid/Zivid.h>
+
+#include <iostream>
+
+int main()
+{
+    try
+    {
+        Zivid::Application zivid;
+
+        std::cout << "Connecting to the camera" << std::endl;
+        auto camera = zivid.connectCamera();
+
+        const size_t captures = 3;
+        const size_t framesPerCapture = 3;
+        for(size_t set = 1; set <= captures; set++)
+        {
+            std::cout << "Configure HDR settings" << std::endl;
```

Comment:
Use same mood all over? Previously you say "connecting", here it should probably be "Configuring".